src/components/JobParameterModal.vue

```diff
@@ -8,7 +8,7 @@
       </ion-buttons>
       <ion-title>{{ translate('Custom Parameters') }}</ion-title>
       <ion-buttons slot="end">
-        <ion-button color="primary" :disabled="currentJob.statusId === 'SERVICE_PENDING'" @click="save()">{{ translate('Save') }}</ion-button>
+        <ion-button color="primary" :disabled="isSaveDisabledByStatus(currentJob.statusId)" @click="save()">{{ translate('Save') }}</ion-button>
```

Comment:
We can instead revert this check to `:disabled="currentJob.statusId !== 'SERVICE_DRAFT'"`, as we only need to enable save button for draft jobs.